releases.yml

```
@@ -1,4 +1,71 @@
 releases:
+  rc.10:
+    assembly:
+      basis:
+        assembly: "rc.9"
+        reference_releases!: {}
+      group:
+        advisories:
+          extras: 138188
+          image: 138189
+          metadata: 138190
+          microshift: 138191
+          rpm: 138193
+          prerelease: 145730
+        operator_index_mode: pre-release
```

```suggestion
          prerelease: 145730
        operator_index_mode: pre-release
```
Is it accurate to keep prelease from rc.9 for rc.10?